ehr_ml/clmbr/__init__.py

```diff
@@ -17,6 +17,7 @@
 from pathlib import Path
 from collections import defaultdict
 from shutil import copyfile
+from tqdm import tqdm
```

Comment:
We aren't actually using this correctly? I see you initialize but never update the progress bar.

Reply:
The progress bar automatically tracks when you start iterating over it, so there's no need to manually update. You can see the final progress bar in the notebooks in piton_private.